src/Meshes/geometry.ts

```diff
@@ -1000,6 +1001,24 @@ export class Geometry implements IGetSetVerticesData {
         return geometry;
     }
 
+    public make_override(id: string): Geometry {

```

Comment:
no need for id
we can get it with "clone" + this.id

I also suggest naming it clone()

Reply:
I'll rename it. It is a bit of a weird clone function...!

Reply:
What if it is cloned many times (as in my example in the PR)? Then clone+this.id will be taken. Not sure if it needs to be unique, there is an uniqueId for that.

Reply:
yes this is supposed to be like that. all the clone will have the same id but different uniqueId